website/docs/components/logo.mdx

```diff
@@ -0,0 +1,11 @@
+import { StorybookComponent } from "@site/src/components/StorybookComponent";
+
+<StorybookComponent />
+
+<StorybookComponent variant="default" implementations={["core", "angular", "react"]} />
+
+<StorybookComponent variant="with label" implementations={["core", "angular", "react"]} />
```

Comment:
De `variant` props zijn de door Storybook gegenereerde story id's. In Docusaurus ziet het er nu 'n beetje angstaanjagend uit 😄 

![image](https://github.com/dso-toolkit/dso-toolkit/assets/9696261/de0eae0d-aa4d-4ade-a913-4284e86cb0cd)

Reply:
Dit gaat nu goed, zie https://www.dso-toolkit.nl/_2265-new-logo-component/components/logo